src/diffusers/models/attention_processor.py

```diff
@@ -56,7 +56,8 @@ def __init__(
         bias=False,
         upcast_attention: bool = False,
         upcast_softmax: bool = False,
-        cross_attention_norm: bool = False,
+        cross_attention_norm: Union[bool, str] = False,
```

Comment:
I'd prefer to not do both a boolean or string, but this is the simplest method to upgrade the flag to support multiple layer types